articles/logs/streams/health.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Log Streams Health
+description: All Log Streams types let you view your stream health in the health tab.
+toc: false
+topics:
+ - logs
+ - streams
+ - event-streams
+contentType: how-to
+---
+
+# HTTP Event Log Streams
+
+HTTP Event Log Streams let you export your log events to the server or target of your choice. When Auth0 creates a log entry for your tenant, a copy is automatically sent to a given URL via an HTTP POST request.
+
+If you use Amazon Web Services, Auth0 also offers an [AWS EventBridge integration](/integrations/aws-eventbridge).
+
```

Comment:
Before we dive into the how, maybe give a quick overview of what is log stream health.

# What is Log Stream Health

Log Stream Health is the functionality that allows customers to troubleshoot issues with Log Streams. It allows customers to get the status of the latest delivery attempts of each stream to make it easy to identify problems in the delivery.

# How to use Stream Health

** add some stuff here**

Status types:

| Status | Description |
|---------|-------------|
| Active (with no errors) | This means that the latest log deliveries have been successful |
| Active (with errors) | The log deliveries are active but the stream failed to deliver logs at one or more points |
| Paused | The customer has pausued the stream. |
| Suspended | Our system has disabled the stream because there is an issue and your logs could not be delivered. You may restart the stream by following the directions or returning to list view and clicking the dropdown option "restart" |